src/pages/thank-you-page/ui/OrderSummary/OrderSummary.module.scss

```diff
@@ -0,0 +1,78 @@
+.summary {
+  margin-top: 20px;
+  font-size: 14px;
+  border-radius: 8px;
+  padding: 20px;
+}
+
+h3 {
+  font-size: 25px;
+  margin-bottom: 15px;
+  color: #333;
+  font-family: 'Helvetica', sans-serif;
+}
+
+.itemList {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+  border-bottom: 1px solid #eaeaea;
+}
+
+.itemName {
+  font-weight: bold;
+  font-size: 16px;
+  color: #444;
+  font-family: 'Helvetica', sans-serif;
```

Comment:
this style already has body (parent) element 
Child elements inherit from their parent elements. That is, if the parent element body has a text color of red, then all elements in body will also have a text color of red. That is why these are "cascading" style sheets.